docs/_guides/gke.md

```diff
@@ -34,6 +34,13 @@ $ kubectl config current-context
 gke_dispatch-193801_us-west1-c_dispatch-demo
 ```
 
+## Setup Docker
+
+In order to build new images to GKE, docker must have the right credentials. This should handle that.
```

Comment:
This is only needed if you are building images or using the google container registry.